Configuration/StandardSequences/python/Services_cff.py

```diff
@@ -14,3 +14,9 @@ def _addCUDAServices(process):
 
 from Configuration.ProcessModifiers.gpu_cff import gpu
 modifyConfigurationStandardSequencesServicesAddCUDAServices_ = gpu.makeProcessModifier(_addCUDAServices)
+
+# load TritonService when SONIC workflow is enabled
+def _addTritonService(process):
+	process.load("HeterogeneousCore.SonicTriton.TritonService_cff")
+from Configuration.ProcessModifiers.enableSonicTriton_cff import enableSonicTriton
+modifyConfigurationStandardSequencesServicesAddTritonService_ = enableSonicTriton.makeProcessModifier(_addTritonService)
```

Comment:
In principle loading of `TrigonService_cff` could be done in the customize function, but to me doing it here with a Modifier does not make a big difference. (and now it demonstrates how "adding more Services like these" would look like)

(this is my last comment)

Reply:
I'd prefer to keep it here if that's acceptable.